extras.php

```diff
@@ -1,12 +1,12 @@
 <?php
-
 /**
- * Extra File where a lot of the extra functions from plugin.php go.
+ * Extra File
+ *
+ * Contains extra functions from plugin.php go.
  *
+ * @since 4.4.0
```

Comment:
This is actually incorrect, because the file isn't included in WordPress core, and the plugin isn't at 4.4.0